components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -828,7 +834,7 @@ pub fn main() {
                         match cmd {
                             DeploymentEvent::Interrupted(message) => {
                                 println!(
-                                    "{} Error publishing transactions: {}",
+                                    "{} Error publishing transactions (x): {}",
```

Comment:
Why this change?
I don't picture how it will look like

Reply:
Addressed with latest